cmd/telemetrygen/internal/common/config.go

```diff
@@ -56,6 +56,8 @@ type Config struct {
 	HTTPPath           string
 	Headers            KeyValue
 	ResourceAttributes KeyValue
+	SignalAttributes   KeyValue
```

Comment:
As these attributes apply to the telemetry data being transmitted (metric datapoints, spans, log records) what would you think of the name `TelemetryAttributes` instead?